aspnetcore/fundamentals/metapackage.md

```diff
@@ -60,3 +60,25 @@ Any dependencies of the preceding packages that otherwise aren't dependencies of
 
 * `StackExchange.Redis` as a dependency of `Microsoft.Extensions.Caching.Redis`
 * `Microsoft.ApplicationInsights` as a dependency of `Microsoft.AspNetCore.ApplicationInsights.HostingStartup`
+
+## Update ASP.NET Core 2.1
+
+We recommend you migrate to the `Microsoft.AspNetCore.App` metapackage. To Update `Microsoft.AspNetCore.App` to ASP.NET Core 2.1:
+
+* On development machines and build servers: Install the latest [.NET Core SDK](https://www.microsoft.com/net/download).
+* On deployment servers: Install the latest [.NET Core runtime](https://www.microsoft.com/net/download).
+
+Your app will roll forward to the latest installed version on an application restart.
+
+## Update ASP.NET Core 2.0
```

Comment:
I believe 2.0 is going out of support in a few days, so I'm not sure if we really need this.